code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -295,7 +295,7 @@ var/bomb_set
 	var/off_station = 0
 	var/turf/bomb_location = get_turf(src)
 	if( bomb_location && (bomb_location.z == map.zMainStation) )
-		if( (bomb_location.x < (128-NUKERANGE)) || (bomb_location.x > (128+NUKERANGE)) || (bomb_location.y < (128-NUKERANGE)) || (bomb_location.y > (128+NUKERANGE)) )
+		if( (bomb_location.x < (256-NUKERANGE)) || (bomb_location.x > (256+NUKERANGE)) || (bomb_location.y < (256-NUKERANGE)) || (bomb_location.y > (256+NUKERANGE)) )
```

Comment:
Use world.maxx and world.maxy instead, so that this issue doesn't appear again when we implement 2048x2048 maps